apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/EnumMap.cs

```diff
@@ -30,14 +30,26 @@ public ApiValueAttribute(string value)
             Value = value;
         }
 
-        internal static void ForEachStaticField<T>(Action<T, string> execute)
+        public static IEnumerable<ApiValueName<T>> GetApiValueNamesIn<T>() where T : struct => 
```

Comment:
I'd make this and the struct (if we keep it) both internal. Even though the class is internal anyway, it's easier to observe that it's internal if you don't need to look at the containing class declaration.

Reply:
OK